raiden/network/transport/matrix/utils.py

```diff
@@ -245,12 +261,23 @@ def _presence_listener(self, event: Dict[str, Any]):
             # Cached presence state matches, no action required
             return
 
+        self.log.debug(
+            "Changing user presence state",
+            user_id=user_id,
```

Comment:
Is this different from `self._user_id`? Otherwise it's already included in the bound logger.

Reply:
Yes it's the user_id whose presence has changed. The bound variable is `current_user` (i.e. the user the trnasport is currently logged in as).